### PR TITLE
fix(working-with-json-fields): Restructure

### PR DESCRIPTION
**Pull request by janpio**
- Split advanced and simple filtering (because only "advanced" used to be a preview feature and is also more complex)
- Restructure introduction of advanced section (Actual info first, then warning, and finally version info)

### CODE REVIEW

**Comment by janpio** [comment]
Internal discussion: https://prisma-company.slack.com/archives/C5Z9TH6N9/p1686599254481689

File: content/200-concepts/100-components/02-prisma-client/051-working-with-fields/100-working-with-json-fields.mdx [hunk 1]
```diff
@@ -175,9 +144,48 @@ const getUsers = await prisma.user.findMany({
...
 
+## Filter on a <inlinecode>Json</inlinecode> field (advanced)
+
+You can also filter rows by the data inside a `Json` field. We call this **advanced `Json` filtering**. This functionality is supported by [PostgreSQL](/concepts/database-connectors/postgresql) and [MySQL](/concepts/database-connectors/mysql) only with [different syntaxes for the `path` option](#path-syntax-depending-on-database). 
+
+<Admonition type="warning">
+
+PostgreSQL does not support [filtering on object key values in arrays](#filtering-on-object-key-value-inside-array).
+
+</Admonition>
```
Comment (ruheni) [inline content/200-concepts/100-components/02-prisma-client/051-working-with-fields/100-working-with-json-fields.mdx]: Is this still accurate?

**Comment by ruheni** [comment]
Let's get the Preview deployment working.

File: content/200-concepts/100-components/02-prisma-client/051-working-with-fields/100-working-with-json-fields.mdx (hunk 1 above)
Comment (janpio) [reply content/200-concepts/100-components/02-prisma-client/051-working-with-fields/100-working-with-json-fields.mdx]: No idea. It is in the current content, so I left it there.

**Comment by janpio** [comment]
Can you help with that @ruheni? To me the content looks fine and the error messages are in no way helpful.

**Comment by ruheni** [comment]
Sure! I'll pick it up later today.

**Review by ruheni — APPROVED** [review]

**Comment by janpio** [comment]
Still not building on Vercel 😢

**Comment by ruheni** [comment]
I spent a little more time than I should trying to figure it out. It doesn't work locally either. I'll give it another stab.

**Comment by jharrell** [comment]
@janpio shall we take another crack at this? 🙂

**Comment by janpio** [comment]
Yes, I think that would still be valuable. I assume the intention behind this change is still possible with the new docs, so please take over.

**Pull request merged**